docs/index.rst

```diff
@@ -67,6 +67,16 @@ Or run the full checks with tox:
     $ tox
 
 
+Upgrading
+---------
+
+The following upstream releases are used to upgrade Bootstrap Flask:
+- `Bootstrap <https://github.com/twbs/bootstrap/releases>`_
+- `Bootstrap Icons <https://github.com/twbs/icons/releases>`_
+- `Bootswatch <https://github.com/thomaspark/bootswatch/tags>`_
+- `Floating UI <https://github.com/floating-ui/floating-ui/releases>`_
```

Comment:
What is it?

Reply:
This is for maintainers that want to do an upgrade to quickly see where to get the latest version to upgrade of what is shipped with Bootstrap-Flask.

Reply:
The doc is for users. You can create a wiki page for content like this.

Reply:
Rmoved to add to wiki